packages/components/src/mobile/global-styles-context/utils.native.js

```diff
@@ -143,6 +143,7 @@ export function getBlockTypography(
 	const typographyStyles = {};
 	const customBlockStyles = blockStyleAttributes?.style?.typography || {};
 	const blockGlobalStyles = baseGlobalStyles?.blocks?.[ blockName ];
+	const parsedFontSizes = Object.values( fontSizes ?? {} );
```

Comment:
This comes in through an externally exposed API, so we're properly handling objects and nullish values. 

It's used twice below, which is why we've moved it here.